ext/intl/grapheme/grapheme_string.c

```diff
@@ -918,4 +918,185 @@ PHP_FUNCTION(grapheme_str_split)
 	ubrk_close(bi);
 }
 
+PHP_FUNCTION(grapheme_levenshtein)
+{
+	zend_string *string1, *string2;
+	zend_long cost_ins = 1;
+	zend_long cost_rep = 1;
+	zend_long cost_del = 1;
+
+	ZEND_PARSE_PARAMETERS_START(2, 5)
+		Z_PARAM_STR(string1)
+		Z_PARAM_STR(string2)
+		Z_PARAM_OPTIONAL
+		Z_PARAM_LONG(cost_ins)
+		Z_PARAM_LONG(cost_rep)
+		Z_PARAM_LONG(cost_del)
+	ZEND_PARSE_PARAMETERS_END();
+
+	if (cost_ins <= 0 || cost_ins > UINT_MAX / 4) {
```

Comment:
I played with your new feature, it looks solid. However, what I would like is to add more tests to trigger those cases.

Reply:
I added corner case. Please let me know if there are any tests that I should increase.